src/classes/StringCreator.java

```diff
@@ -0,0 +1,33 @@
+package classes;
+
+import java.util.Random;
+
+/**
+ * Created by aloginov on 13.09.16.
+ */
+public class StringCreator {
```

Comment:
Interesting idea. But read about patterns. For example, builder, factory, etc. A book "GoF"